projects/design-system-initiafy/src/lib/button/button.component.spec.ts

```diff
@@ -176,6 +176,39 @@ describe('ButtonComponent', () => {
 
   });
 
+  describe('When testing design', () => {
```

Comment:
what is this test for? When testing design? What does that mean? Maybe it should be named "Button Design"? If errors pop up in the karma report it should be immediately  obvious where the error is. If I see an issue with 'When testing design', I don't think I would know to look in the Button.spec file. Can we at least put the word button in the string?

Reply:
This "**describe('When testing design', () =>** " is like a group of unit tests; So when we have an error in one of its children tests, it would show like **[parent name + failed test name]**;
Example: [When testing design should show default design if variable is empty] failed
We can add button to parents names like "When testing button design" and "When testing button default classes"?

Reply:
Oh ok so. Thank you for that